### PR TITLE
Opt-out from regendoc for problematic example

### PR DESCRIPTION
**Pull request by The-Compiler**
See #8807

### CODE REVIEW

File: doc/en/how-to/skipping.rst [hunk 1]
```diff
@@ -371,7 +371,7 @@ Running it with the report-on-xfail option gives this output:
 
 .. code-block:: pytest
 
-    example $ pytest -rx xfail_demo.py
+    ! pytest -rx xfail_demo.py
```
Comment (nicoddemus) [inline doc/en/how-to/skipping.rst:374]: Can you perhaps add a comment outside the block, pointing to a relevant issue in regendoc (and mention this there)? Just so we can eventually change this back to `$` once regendoc gets fixed. 👍

**Review by nicoddemus — APPROVED** [review]

File: doc/en/how-to/skipping.rst (hunk 1 above)
Comment (The-Compiler) [reply doc/en/how-to/skipping.rst:374]: IMHO, those kind of FIXME comments as part of code always get lost/outdated anyways - given that we already have #8807 and the commit message points to it, that might be enough already?

Comment (nicoddemus) [reply doc/en/how-to/skipping.rst:374]: No worries, but personally I see value in them: whenever I encounter a "TODO" with an issue link, I always click on it and see if it has been resolved and the TODO warrants a review. I've found many "forgotten" TODOs that way.

Plus when someone sees that weird `!` there, a link will promptly explain why that's there.

But as I said, I don't feel too strongly either, so feel free to leave it out if you think it is not necessary. 👍

Comment (The-Compiler) [reply doc/en/how-to/skipping.rst:374]: Ok, done!

**Pull request merged**